ui/apps/platform/src/Containers/Collections/CollectionAttacher.tsx

```diff
@@ -58,7 +60,7 @@ function CollectionAttacher({
                 rowKey={({ id }) => id}
                 cells={collectionTableCells}
                 selectedLabel="Attached collections"
-                deselectedLabel="Detached collections"
+                deselectedLabel="Available collections"
                 selectButtonText="Attach"
                 deselectButtonText="Detach"
```

Comment:
Although attach versus detach still works, possible discussion item to use more ordinary terms:
* Attached > Selected
* Attach > Add
* Detach > Remove